src/gps/read_usb_gps.cpp

```diff
@@ -0,0 +1,63 @@
+
+#include <libgpsmm.h>
+
+#include "../log.h"
+#include "../simulator/world_interface.h"
+#include "../simulator/utils.h"
+
+// UW lat/long is 47.653116, -122.305619
+// source: http://www.csgnetwork.com/degreelenllavcalc.html
+const double METERS_PER_DEG_NS = 111183.53599983045;
```

Comment:
Should we put these in `Constants.h`?

Reply:
Also, it might be a better idea to use constexpr for these since they're primitive types.

Reply:
Unless we're planning to use these constants elsewhere in the codebase, I think it's easier to keep the constants here, close to where they're used. I've been meaning to move some of our other constants _out_ of `Constants.h`, actually.